core/src/edu/cornell/gdiac/main/BouncePlatformModel.java

```diff
@@ -46,6 +49,8 @@ public float getDefaultMaxSpeed() {
     return defaultMaxSpeed;
   }
 
+  public void setOffsetX(int offset) { offsetX = offset; }
```

Comment:
This method should not be public since you will be only calling it in the initialization of the Bounce Platform Model when making your changes.